docs/source/operations/projections/airocean.rst

```diff
@@ -0,0 +1,57 @@
+.. _airocean:
+
+********************************************************************************
+Airocean
+********************************************************************************
+
+The Airocean map projection is based on the Dymaxion Fuller projection, which is a icosahedron polyhedral projection that intends
```

Comment:
we should probably remove the Dymaxion word here...

Reply:
Seems fair.. 
I also spotted a typo as well for the word _cubostahedron_.

It seems like CI checks are failing, but the file concerned (mod_ster.cpp) is not part of this pull request. Did I integrated commits ahead of the current master branch?

Reply:
hum I just saw an issue with the appveyor target that complained about the sqrtl() usage, that should be replace with sqrt() (without l)
```
..\src\projections\airocean.cpp(187): error C2220: warning treated as error - no 'object' file generated
..\src\projections\airocean.cpp(187): warning C4244: 'initializing': conversion from 'long double' to 'double', 
```

Reply:
hum there's a unrelated issue with -Wshadow that I've just fixed in https://github.com/OSGeo/PROJ/pull/4304. Please rebase your pull request on top of master, once PR #4304 has been merged into master

Reply:
PR #4304 has been merged into master, so you can rebase on top of latest HEAD

Reply:
I'm on it

Reply:
There. It should be synced up with master now.

I altered the doc as you wanted.